stl/inc/xutility

```diff
@@ -2147,26 +2148,30 @@ namespace ranges {
         inline constexpr _Begin::_Cpo begin;
     }
 
-    // CUSTOMIZATION POINT OBJECT ranges::end
+    // ALIAS TEMPLATE ranges::iterator_t (Implements D2091R0)
+    template <class _Ty>

```

Comment:
Note the template parameter is now `class _Ty` instead of `range _Rng` (which change is somewhat lost in the relocation). Now that `iterator_t` is unconstrained, we can use it to simplify the definition of `ranges::end` (Lines 2169 and 2174) so it's relocated here.